From ba9c901b58d3d9b55e0bc2e8e5ee4549dcdf6847 Mon Sep 17 00:00:00 2001
From: Votond <votond@outlook.com>
Date: Mon, 23 Feb 2026 01:02:54 +0300
Subject: [PATCH] test

---
 library/File.d.lua | 1 +
 1 file changed, 1 insertion(+)

diff --git a/library/File.d.lua b/library/File.d.lua
index dd2948d..872012d 100644
--- a/library/File.d.lua
+++ b/library/File.d.lua
@@ -24,6 +24,7 @@ function File.GetModSize() end
 ---@return any
 function File.DoString(code, lua) end
 
+---@overload fun(relativePath: string): any
 ---@param relativePath string
 ---@param lua Script
 ---@return any
-- 
2.43.0

